src/env-inl.h

```diff
@@ -221,6 +221,13 @@ inline Environment::Environment(v8::Local<v8::Context> context,
   set_as_external(v8::External::New(isolate(), this));
   set_binding_cache_object(v8::Object::New(isolate()));
   set_module_load_list_array(v8::Array::New(isolate()));
+
+  v8::Local<v8::FunctionTemplate> fn_t = v8::FunctionTemplate::New(isolate());
+  fn_t->SetClassName(FIXED_ONE_BYTE_STRING(isolate(), "InternalFieldObject"));
+  v8::Local<v8::ObjectTemplate> obj_t = fn_t->InstanceTemplate();
+  obj_t->SetInternalFieldCount(1);
+  set_generic_internal_field_template(obj_t);
```

Comment:
Mild cognitive dissonance here, the `_t` suffix makes it look like the variables are typedefs at a quick glance.